### ISSUE TITLE
karabiner-elements on arm mac m1 

### ISSUE THREAD

**Issue by hackftz** [pqrs-org/Karabiner-Elements#2577]
I use the new arm mac mini m1, when I install the karabiner-elements app, I can not start the macos (big sur) system normally, It always causes system crash.

**Comment by MuhammedZakir** [comment]
Related: https://github.com/pqrs-org/Karabiner-Elements/issues/2513.

**Comment by hackftz** [comment]
my macos version is big sur 11.1 (20C69)

**Comment by EnricoTirotta** [comment]
Same here. M1 restarts always with a Kernel Panic. Without Karabiner M1 goes fine.

**Comment by matsuda0707** [comment]
I have the same problem on my macmini 2020 (M1 chip). The karabiner-elements still releases this version (work on Intel version,but NOT work on M1)
Please fix this as soon as possible!

**Comment by hackftz** [comment]
yes, please help me, I need it to make my work easier with my hhkb. 😄

**Comment by damiencollard** [comment]
I'm interested too!

**Comment by wardz** [comment]
Same issue here.

**Comment by ruiaraujo012** [comment]
Same here

**Comment by ferris1993** [comment]
Same here.

**Comment by pquimo** [comment]
tekezo, I don't know if you have an M1 to test on, but if you need help gathering more information on this issue, I'd be happy to help, as it is happening for me as well.  Thank you.

**Comment by tekezo** [comment]
This issue was caused by an issue of macOS Big Sur kernel on Apple Silicon that is caused by using DriverKit.
Karabiner-Elements is using DriverKit and the usage invokes the kernel issue and causes kernel panic, and there was no practical workaround to avoid the issue.

However, this macOS problem has been fixed in macOS Big Sur 11.3 Beta!

There is no panics on M1 macs with the latest beta of macOS Big Sur.
https://beta.apple.com/sp/betaprogram/
